src/utility/index.jsx

```diff
@@ -73,22 +75,46 @@ export function fromNow(input) {
   // Diff the current and input timestamps in seconds
   const diff = (now.getTime() - date.getTime()) / 1000;
 
+  let fromNow;
+
   if (diff < 0) {
-    return 'in the future';
+    fromNow = 'in the future';
   } else if (diff < 2) {
-    return 'just now';
+    fromNow = 'just now';
   } else if (diff < (minute * 2)) {
-    return `${diff.toFixed(0)} seconds ago`;
+    fromNow = `${diff.toFixed(0)} seconds ago`;
   } else if (diff < (hour * 2)) {
-    return `${(diff / minute).toFixed(0)} minutes ago`;
+    fromNow = `${(diff / minute).toFixed(0)} minutes ago`;
   } else if (diff < (day * 2)) {
-    return `${(diff / hour).toFixed(0)} hours ago`;
+    fromNow = `${(diff / hour).toFixed(0)} hours ago`;
   } else if (diff < (month * 2)) {
-    return `${(diff / day).toFixed(0)} days ago`;
+    fromNow = `${(diff / day).toFixed(0)} days ago`;
   } else if (diff < (year * 2)) {
-    return `${(diff / month).toFixed(0)} months ago`;
+    fromNow = `${(diff / month).toFixed(0)} months ago`;
+  } else {
+    fromNow = `${(diff / year).toFixed(0)} years ago`;
+  }
+
+  if (tooltip) {
```

Comment:
I think the string logic and the tooltip generation should be separated into different functions.